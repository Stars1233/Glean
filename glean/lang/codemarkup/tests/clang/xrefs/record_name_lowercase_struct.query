query: cxx1.RecordStructLowerCase _
transform: [gensort, []]
