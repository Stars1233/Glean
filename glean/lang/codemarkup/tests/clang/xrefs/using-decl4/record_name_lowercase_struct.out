[
  "@generated",
  { "key": { "name_lowercase": "bar", "name": "Bar" } },
  { "key": { "name_lowercase": "foo", "name": "Foo" } },
  { "key": { "name_lowercase": "xxx", "name": "XXX" } },
  { "key": { "name_lowercase": "yyy", "name": "YYY" } },
  { "key": { "name_lowercase": "zzz", "name": "ZZZ" } }
]